sys/include/net/gnrc/netif/conf.h

```diff
@@ -83,11 +83,13 @@ extern "C" {
 /**
  * @brief   Maximum number of multicast groups per interface
  *
- * Default: 2 (all-nodes + solicited-nodes of link-local and global unicast
+ * Default: 3 (all-nodes + solicited-nodes of link-local and global unicast
  * address) + @ref GNRC_NETIF_RPL_ADDR + @ref GNRC_NETIF_IPV6_RTR_ADDR
  */
 #ifndef GNRC_NETIF_IPV6_GROUPS_NUMOF
-#define GNRC_NETIF_IPV6_GROUPS_NUMOF   (2 + GNRC_NETIF_RPL_ADDR + GNRC_NETIF_IPV6_RTR_ADDR)
+#define GNRC_NETIF_IPV6_GROUPS_NUMOF   (GNRC_NETIF_IPV6_ADDRS_NUMOF + \
```

Comment:
Could you elaborate on why the number of groups is now depending on the number of overall IPv6 addresses (`GNRC_NETIF_IPV6_ADDRS_NUMOF`)?

Reply:
Because this way it doesn't matter to which number `GNRC_NETIF_IPV6_ADDRS_NUMOF` is set by the user. It will always work.

Reply:
(the doc above still applies, even more than before ;-))

Reply:
Okay, since we are still able to set the number of groups explicitly and independent from `GNRC_NETIF_IPV6_ADDRS_NUMOF`, it is not such a big deal. It increases the friendliness as @jcarrano pointed out.